tests/testthat/helper-mcstate2.R

```diff
@@ -1,8 +1,15 @@
 ex_simple_gamma1 <- function(shape = 1, rate = 1) {
-  mcstate_model(
-    parameters = "gamma",
-    direct_sample = function(rng) rng$gamma(1, shape = shape, scale = 1 / rate),
-    density = function(x) dgamma(x, shape = shape, rate = rate, log = TRUE),
-    gradient = function(x) (shape - 1) / x - rate,
-    domain = rbind(c(0, Inf)))
+  e <- new.env(parent = .GlobalEnv)
```

Comment:
This bit of butchery won't be obvious, but is in order to prevent a warning that "namespace mcstate may not be available on load" while serialising for parallel use.